<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TCRsettingDialog</class>
 <widget class="QDialog" name="TCRsettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>541</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>温度条件设定</string>
   </property>
   <widget class="QPushButton" name="Tempsave_pushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>510</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>阶段1：设定温度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="stage1stemp_lineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>℃ 阶段1：计算温度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="stage1ctemp_lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>℃ </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>510</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>阶段3：设定温度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="stage3stemp_lineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>℃ 阶段1：计算温度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="stage3ctemp_lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>℃ </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>510</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>阶段2：设定温度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="stage2stemp_lineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>℃ 阶段1：计算温度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="stage2ctemp_lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>℃ </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>541</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>安定时间设置</string>
   </property>
   <widget class="QPushButton" name="Stimesave_pushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>478</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>安定时间1：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="steadytime1_lineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>分钟</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>安定时间2：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="steadytime2_lineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>分钟</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>541</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>超时时间设置</string>
   </property>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>658</x>
      <y>31</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>分钟</string>
    </property>
   </widget>
   <widget class="QPushButton" name="otimesave_pushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>472</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>降温超时：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Dtime_lineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>分钟</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>升温超时1：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Utime1_lineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>分钟</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>237</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>升温超时2：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Utime2_lineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>分钟</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="close_pushButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="print_checkBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>结束后自动打印</string>
   </property>
  </widget>
  <widget class="QPushButton" name="printsave_pushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
